packages/synapse-react-client/src/components/ReleaseCard/ReleaseCardLarge.tsx

```diff
@@ -73,22 +79,26 @@ export const ReleaseCardLarge: React.FunctionComponent<
             <CalendarTodayTwoTone sx={{ color: 'grey.700' }} />
             <Typography variant="headline1">
               {releaseCardConfig.prependRelease && 'Release '}
-              {releaseVersion}
+              {releaseName}
             </Typography>
           </Stack>
           <Stack
             direction={{ xs: 'column', sm: 'row' }}
             alignItems={{ xs: 'flex-start', sm: 'center' }}
             gap="20px"
           >
-            {buttonToExplorePageConfig && pathAndQuery && (
-              <Button variant="contained" color="primary" href={pathAndQuery}>
-                {buttonToExplorePageConfig.label}
+            {primaryBtnConfig && primaryBtnPathAndQuery && (
+              <Button
+                variant="contained"
+                color="primary"
+                href={primaryBtnPathAndQuery}
+              >
+                {primaryBtnConfig.label}
               </Button>
             )}
-            {releaseCardConfig.dataGuidePath && (
-              <Link href={releaseCardConfig.dataGuidePath}>
-                View Data Guide
+            {secondaryBtnConfig && secondaryBtnPathAndQuery && (
+              <Link href={secondaryBtnPathAndQuery}>
+                {secondaryBtnConfig.label}
```

Comment:
Secondary button should also link to explore page with selected facets, rather than to a static path